nuntium/popit_api_instance.py

```diff
@@ -29,9 +54,15 @@ def fetch_all_from_api(cls, instance, writeitinstance):
             PopitPerson.create_contact(obj, doc, writeitinstance)
 
     @classmethod
+    def determine_if_person_is_current(cls, doc):
```

Comment:
This could actually be a function rather than a class method too. I think putting all this on the class perhaps just complicates things - you can imagine a class you wanted to mix this in to already having an is_active() method on it. I would just have a couple of functions:

```
def is_current_membership(membership_doc):
    ...

def is_current_person(person_doc):
    return any(is_current_membership(membership) for membership in person_doc.get('memberships', ()))
```